Dockerfile

```diff
@@ -1,8 +1,6 @@
 # ---- Base image ----
-FROM node:8.12.0-slim as base
+FROM hmcts.azurecr.io/hmcts/base/node/stretch-slim-lts-8:latest as base
```

Comment:
Seems Codacy is not happy with the `latest` tag, which is legit. However this base image is aimed at being updated with patch-only version (this approach is chosen for automatic security patches deployments). The nodeJS major version will never change.